internal/indexnode/index.go

```diff
@@ -56,13 +56,13 @@ func (index *CIndex) Serialize() ([]*Blob, error) {
 	var cBinary C.CBinary
 
 	status := C.SerializeToSlicedBuffer(index.indexPtr, &cBinary)
-	defer C.DeleteCBinary(cBinary)
```

Comment:
another soluation
defer func(){
   if C.CBinary is empty{
     return 
  }
 C.DeleteBinary(cBinary)
}